### ISSUE TITLE
Vakeurs par défaut des Difficultés Cirkwi

### ISSUE THREAD

**Issue by babastienne** [GeotrekCE/Geotrek-admin#4308]
D'après un échange aujourd'hui avec Cirkwi j'ai eu les infos comme quoi l'échelle de difficulté côté Cirkwi est la suivante : 
- 1 et 2 : très facile
- 3 et 4 : facile
- 5 et 6 : moyen
- 7 et 8 : difficile

Actuellement dans les fixtures Geotrek qui servent à créer automatiquement les Difficultés et leur niveau Cirkwi associé lors de l'installation on a les valeurs suivantes : 
- Très facile = 1
- Facile = 2
- Moyen = 3
- Difficile = 4
- Très difficile = 5

Pour coller avec leur échelle je propose de plutôt avoir les valeurs par suivante dans les fixtures : 
- Très facile = 1
- Facile = 3
- Moyen = 5
- Difficile = 7
- Très difficile = 8

Il faut modifier ce morceau de code : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/trekking/fixtures/basic.json#L419

**Comment by camillemonchicourt** [comment]
Oui. Si Cirkwi a fait évoluer ses typologies, ça vaut le coup de le répercuter dans les correspondances dans Geotrek-admin.
Mais à voir si on ne peut le faire que pour les nouvelles installations, ou aussi mettre à jour les instances existantes. Là ça peut être plus complexe, voire coincer si les niveaux de difficulté ont été revus.
Mais ça serait embêtant d'avoir des instances avec des référentiels différents...

De manière plus globale, il y a un sujet récurrent sur les niveaux de difficulté.
On a souvent les mêmes, mais pas tout à fait, pas tout le temps et c'est une grosse difficulté pour les échanges de données et les passerelles.

Dans Geotrek-admin, on créé en effet 5 niveaux de difficulté par défaut (https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/trekking/fixtures/basic.json#L419-L487) mais finalement au PNE on a préféré se limiter à 4 niveaux. 
Cirkwi en a 4 aussi, Apidae et Visorando en ont 5, Decathlon Outdoor et Outdooractive en ont 3, etc...
Il y a un vrai enjeu à standardiser cela car c'est un point central dans le partage de données.
Nous avons échangé avec le FFRP sur le sujet qui aurait la légitimité pour porter cette reflexion, mais ils semblent avoir retenu un système encore plus complexe avec 3 critères (https://www.ffrandonnee.fr/Media/Default/Documents/randonner/Guide-cotation-ffrandonnee-2020.pdf).

Pour nous, cela un enjeu fort du travail sur la standardisation des données que nous allons mener à partir de 2025 (https://www.ecrins-parcnational.fr/contractuel-cat-chargee-mission-open-data-outdoor-biodiversite).

**Comment by babastienne** [comment]
Je trouve l'idée de séparer la difficulté en trois critères (risque, technicité, effort) plutôt pertinente et plus précise que les difficultés actuellement proposées par défaut dans Geotrek. On pourrait proposer ça par défaut via les échelles de cotation.

**Comment by camillemonchicourt** [comment]
> Je trouve l'idée de séparer la difficulté en trois critères (risque, technicité, effort) plutôt pertinente et plus précise que les difficultés actuellement proposées par défaut dans Geotrek. On pourrait proposer ça par défaut via les échelles de cotation.

Les retours sont très mitigés sur le sujet. C'est trop complexe pour les utilisateurs qui veulent une info claire, simple et lisible : C'est facile ou moyen ou difficile.

Donc l'idée est plutôt de faire simple et pas partir sur différents critères et échelles de cotation.
